test_elasticsearch/test_client/common.py

```diff
@@ -36,15 +36,8 @@ def perform_request(self, method, url, params=None, headers=None, body=None):
         return resp
 
 
-class DummyTransportTestCase:
-    def setup_method(self, _):
-        self.client = Elasticsearch(transport_class=DummyTransport)
-
-    def assert_call_count_equals(self, count):
-        assert count == self.client.transport.call_count
-
-    def assert_url_called(self, method, url, count=1):
-        assert (method, url) in self.client.transport.calls
-        calls = self.client.transport.calls[(method, url)]
-        assert count == len(calls)
-        return calls
+def assert_helper(client, method, url, count=1):
```

Comment:
I think I prefer having a `DummyTransportTestCase` over the `assert_helper()` because it's clear that the `DummyTransportTestCase` knows about the `DummyTransport` class.

Reply:
I don't think this review comment is resolved yet, I still see usages of `assert_helper()`

Reply:
Its part of `DummyTransportTestCase` Now we don't pass the client explicitly to it. 
```python
class DummyTransportTestCase:
    def setup_method(self, _):
        self.client = Elasticsearch(transport_class=DummyTransport)

    def assert_helper(self, method, url, count=1):
        calls = self.client.transport.calls
        assert (method, url) in calls
        assert count == len(calls[(method, url)])
        return calls[(method, url)]
```